### PR TITLE
Mockdata

### PR DESCRIPTION
**Pull request by hewonjeong**
- add 6 item mock data

### CODE REVIEW

File: src/server/database.js [hunk 1]
```diff
@@ -15,3 +15,100 @@ export const clearDB = (cb) => {
...
+      lng: -122.4233365122,
+      address: '310 Dolores St, San Francisco, CA 94110, USA',
+      createdDate: makeTime(-1),
+      modifiedDate: makeTime(-1),
+      category: 'event',
+      startTime: makeTime(-1),
+      endTime: makeTime(1)
+    },
+    {
+      title: 'Generation Beauty by ipsy 2016',
```
Comment (mun625) [inline src/server/database.js:45]: Nice mock data.
They have various location and category, good good.

**Comment by festinalente91** [comment]
LGTM

File: src/server/index.js [hunk 2]
```diff
@@ -4,10 +4,14 @@ import config from 'config';
 import reactRoutes from './react-routes';
 import apiRoutes from './routes/api-routes';
 import bodyParser from 'body-parser';
-
+import {initMock} from './database';
 export default (cb) => {
   const app = express();
-
```
Comment (festinalente91) [inline src/server/index.js:10]: It would be better to add an comment that "remove this before release"

**Comment by festinalente91** [comment]
LGTM

**Comment by mun625** [comment]
LGTM

**Pull request merged**